backend/src/chat/chat.controller.ts

```diff
@@ -50,4 +59,11 @@ export class ChatController {
 	async leaveChannel(@Param('channelId') channelId: string, @Req() req) {
 		return await this.chatService.leaveChannel(+channelId, req.user.sub);
 	}
+
+	@UseGuards(AuthGuard)
+	@Post('channel/update/')
```

Comment:
This route is not explicit enough, and why it is a POST when it should be a GET ?
```suggestion
	@Get('messages/:channelId')
```

Reply:
The goal is toi have the channel name in the body of a POST request, to be able to check it with a DTO.